files/image_config/sudoers/sudoers.lecture

```diff
@@ -0,0 +1,4 @@
+
+Make sure your account has RW permission to current device.
```

Comment:
Suggest reword: `Make sure your account has RW permissions for the current device.`